site/packages/aa-core/accounts/index.md

```diff
@@ -83,7 +83,7 @@ export type SmartContractAccount<Name extends string = string> =
     getInitCode: () => Promise<Hex>;
     isAccountDeployed: () => Promise<boolean>;
     getFactoryAddress: () => Address;
-    getEntrypoint: () => Address;
+    getEntryPoint: () => Address;
```

Comment:
(Not specifically related to 0.7, no change required) I feel like the SDK can be a little inconsistent about when to have a field vs a getter. Seems like at some point when we do some cleanup, we might consider having a convention that we only use getters when it returns a promise, but for plain values just a field e.g. `factoryAddress: Address`.